Code/Engine/Foundation/Profiling/Profiling.h

```diff
@@ -193,17 +193,64 @@ class EZ_FOUNDATION_DLL ezProfilingSystem
 ///
 /// \note The name string must not be destroyed before the current scope ends.
 ///
-/// \sa ezProfilingListScope
+/// \sa nsProfilingListScope
+/// \sa EZ_PROFILE_LIST_NEXT_SECTION
+#    define EZ_PROFILE_LIST_SCOPE(szListName, szFirstSectionName) \
+      nsProfilingListScope EZ_CONCAT(_nsProfilingScope, EZ_SOURCE_LINE)(szListName, szFirstSectionName, EZ_SOURCE_FUNCTION)
+
+/// \brief Starts a new section in a EZ_PROFILE_LIST_SCOPE
+///
+/// \sa nsProfilingListScope
+/// \sa EZ_PROFILE_LIST_SCOPE
+#    define EZ_PROFILE_LIST_NEXT_SECTION(szNextSectionName) nsProfilingListScope::StartNextSection(szNextSectionName)
+
+#  endif
+#  if defined(TRACY_ENABLE)
+#    include <tracy/tracy/Tracy.hpp>
+/// \brief Profiles the current scope using the given name.
+///
+/// It is allowed to nest EZ_PROFILE_SCOPE, also with EZ_PROFILE_LIST_SCOPE. However EZ_PROFILE_SCOPE should start and end within the same list scope
+/// section.
+///
+/// \note The name string must not be destroyed before the current scope ends.
+///
+/// \sa nsProfilingScope
+/// \sa EZ_PROFILE_LIST_SCOPE
+#    define EZ_PROFILE_SCOPE(szScopeName) \
```

Comment:
Is there no way to make use of the user given name?

Reply:
There is a way, using Tracy's ZoneScopedN macro, but this would force some profiling macros to fail, as it wouldn't be able to convert the types for Tracy.

Reply:
But I definitely agree this should be changed.